external/fv3fit/tests/keras/test_pure_keras.py

```diff
@@ -31,38 +27,3 @@ def test_PureKerasDictPredictor_predict():
     in_xr = xarray.Dataset({"a": (["blah", "z"], [[1]])})
     out = predictor.predict(in_xr)
     assert float(out["out"][0]) == pytest.approx(r)
-
-
-@pytest.mark.parametrize(
-    ["unstacked_dims"],
-    [
-        pytest.param((), id="no_unstacked_dims"),
-        pytest.param(("z",), id="column_unstacked_dim"),
-    ],
-)
-def test_PureKerasModel_predict(unstacked_dims):
```

Comment:
Can we keep this test? It seems like it would still be functionality we want tested. Even if in your current code this only calls the helper function, someone else might change that in a later PR.